### PR TITLE
T2.1- Image update PR 2

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

File: samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-5-ConsentPreviouslyGranted.png
Comment (Pawank-MSFT) [inline samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-5-ConsentPreviouslyGranted.png:1]: hide email and other details

File: samples/tab-sso/nodejs/doc/images/tab-sso-details.png
Comment (Pawank-MSFT) [inline samples/tab-sso/nodejs/doc/images/tab-sso-details.png:1]: Hide token

File: samples/tab-sso/nodejs/doc/images/tab-sso-teams.png
Comment (Pawank-MSFT) [inline samples/tab-sso/nodejs/doc/images/tab-sso-teams.png:1]: hide email and other SSO details

**Review by Pawank-MSFT — COMMENTED** [review]
please fix all comments

File: samples/graph-rsc/csharp/RSCDemo/Images/select-people.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/graph-rsc/csharp/RSCDemo/Images/select-people.png:1]: Fixed

File: samples/graph-rsc/nodeJs/Images/select-people.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/graph-rsc/nodeJs/Images/select-people.png:1]: Fixed

File: samples/tab-deeplink/csharp/DeepLinkBot/images/13_Team_Selectpeopleaudio.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/images/13_Team_Selectpeopleaudio.png:1]: Fixed

File: samples/tab-deeplink/csharp/DeepLinkBot/images/18_Team_postclickVedioCall.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-deeplink/csharp/DeepLinkBot/images/18_Team_postclickVedioCall.png:1]: Fixed

File: samples/tab-graph-toolkit/csharp/TabGraphToolkit/Images/8.PersonCard.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-graph-toolkit/csharp/TabGraphToolkit/Images/8.PersonCard.png:1]: Fixed

File: samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-1-NeedsConsent.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-1-NeedsConsent.png:1]: Fixed

File: samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-2-ConsentPopup.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-2-ConsentPopup.png:1]: Fixed

File: samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-3-ConsentDeclined.png
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-sso/nodejs/doc/images/AAD-SSO-Tab-3-ConsentDeclined.png:1]: Fixed

File: samples/tab-sso/nodejs/doc/images/tab-sso-teams.png
Comment (Jegadeesh-MSFT) [reply samples/tab-sso/nodejs/doc/images/tab-sso-teams.png:1]: Fixed

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed Build Pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8892848107

**Pull request merged**